content/documentation/gettingStarted/content.md

```diff
@@ -6,13 +6,24 @@ weight = 120
   sticky = true
 +++
 
-<!--
-TODO:
-* Link for trying it out online
-* Instructions on how to build, run and debug locally
+# Getting Started
 
-➡️ Now as you are up and running, let's have a look at the [Coffee Editor NG architecture]({{< relref  "coffeeeditorng" >}})!
--->
+Welcome to EMF Cloud. Our framework provides two distinct approaches to best accommodate your project's needs. Choose the demo that aligns with your requirements:
 
-We will provide a live demo as well as a link to GitHub as soon as the code is published.
-In the meantime you can have a look at the [Coffee Editor NG architecture]({{< relref  "coffeeeditorng" >}})!
+## Model Hub Approach
+
+Designed for projects that require robust, command-based model management with features like undo/redo, versioning, and persistence. This approach is best suited for users who need a tightly controlled model manipulation environment.
+
+→ Explore the [Model Hub Demo](https://github.com/eclipse-emfcloud/modelhub/tree/main/examples/theia) for a practical example.
+
+## Langium-Based DSL Approach
+
+Tailored for projects that benefit from advanced textual DSL support with integrated Language Server Protocol (LSP) features. This approach excels in delivering a rich editing experience with enhanced language tooling.
+
+→ Check out the [CrossModel Demo](https://github.com/CrossBreezeNL/crossmodel) to see this approach in action.
+
+---
+
+For a detailed comparison of the two approaches and guidance on when to use which, please refer to our [EMF Cloud Approaches Documentation](../coffeeEditorNg/content.md).
```

Comment:
This link is broken, but as mentioned before, I think it'd be better to move up the "Modeling Tool Approaches" page and directly link from there... maybe integrating this getting started page into the tool approaches directly.